django_plastic_tickets/locale/de/LC_MESSAGES/django.po

```diff
@@ -18,11 +18,11 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Poedit 2.4.1\n"
 
-#: forms.py:91
+#: django_plastic_tickets/forms.py:91
```

Comment:
Is there a way to disable these comment updates (or remove these entirely)?
This produces a lot of unnecessary noise.

Reply:
the following can be used in future ```django-admin makemessages --no-location --locale de```

I have deleted all comments for now